src/js/disability-benefits/526EZ/components/DisabilityWizard.jsx

```diff
@@ -0,0 +1,191 @@
+        }
+        {atIncreaseGuidance && sessionStorage.userToken &&
+        <RequiredLoginView
+          containerClass="login-container"
+          authRequired={1}
+          serviceRequired={['disability-benefits']}
+          userProfile={profile}
+          loginUrl={loginUrl}
+          verifyUrl={verifyUrl}>
+          <a className="usa-button-primary" href="/disability-benefits/526/apply-for-increase/introduction/">Apply now<span className="button-icon"> »</span></a>
+        </RequiredLoginView>}
+        {atEbenefitsGuidance &&
+        <a className="usa-button-primary" href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=disability-compensation">Go to eBenefits to Apply<span className="button-icon"> »</span></a>
+        }
+        {currentLayout !== applyGuidance &&
+        <a className="usa-button-primary" onClick={this.goForward}>Next<span className="button-icon"> »</span></a>
+        }
+      </div>);
+    }
+
+  answerQuestion = (field, answer) => {
+    const newState = Object.assign({}, { [field]: answer });
+    this.setState(newState);
+  };
+
+  goToNextPage = () => {
+    let nextLayout = applyGuidance;
+    if (this.state.currentLayout === chooseStatus && this.state.disabilityStatus === 'update') {
+      nextLayout = chooseUpdate;
+    }
+    this.setState({ currentLayout: nextLayout, errorMessage: '' });
+  }
+
+  goBack = () => {
+    let nextLayout = chooseStatus;
+    if (this.state.currentLayout === applyGuidance && this.state.disabilityStatus === 'update') {
+      nextLayout = chooseUpdate;
+    }
+    this.setState({ currentLayout: nextLayout, errorMessage: '' });
+  };
+
+  displayErrorMessage = () => {
+    this.setState({ errorMessage: 'Please select an option' });
+  }
+
+  GetStartedMessage = () => {
+    const { disabilityStatus, add, increase } = this.state;
+    const signInMessage = sessionStorage.userToken ? '' : ' Sign in to your account to get started.';
+    let getStartedMessage = `Based on your answers, you can file a claim for increase.${signInMessage}`;
+    if (disabilityStatus === 'first' || disabilityStatus === 'appeal') {
+      getStartedMessage = 'We currently aren’t able to file an original claim on Vets.gov. Please go to eBenefits to apply.';
+    }
+    if (add && !increase) {
+      getStartedMessage = 'Because you’re adding new conditions, you’ll need to apply using eBenefits.';
+    }
+    if (add && increase) {
+      getStartedMessage = 'Because you have both new and worsening conditions, you’ll need to apply using eBenefits.';
+    }
+    return <p>{getStartedMessage}</p>;
+  }
+
+  goForward = () => {
+    const { currentLayout, disabilityStatus, increase, add } = this.state;
+    const isUpdate = disabilityStatus === 'update';
+    if (currentLayout === chooseStatus && !disabilityStatus) {
```

Comment:
Would it make sense to group some of these together more closely?
```js
if (currentLayout === chooseStatus) {
  if (!disabilityStatus) { return ... }
  if (!increase) { return ... }
  // etc...
}
```
I'm thinking maybe not because these are in a specific order, but figured I should ask.
If they are in this order deliberately, could we add some clarifying comment above this block of control flow explaining why?